deploy/rhsm-clowdapp.yaml

```diff
@@ -1563,6 +1563,53 @@ objects:
                 secretName: pinhead
 
     jobs:
+      #TODO fix all up to liveness probe. not sure what all needs to be changed. updated 7-6
+    - name: rhsm-conduit-cron-sync
+      schedule: ${ORG_SYNC_SCHEDULE}
+      activeDeadlineSeconds: 4200
+      successfulJobsHistoryLimit: 2
+      restartPolicy: Never
+      podSpec:
+       image: ${CURL_CRON_IMAGE}:${CURL_CRON_IMAGE_TAG}
+        command:
+         - /usr/bin/bash
+           - -c
+           - curl
+           - >
+         - curl
+         - curl -X 'POST'   http://rhsm-conduit:8000/api/rhsm-subscriptions/v1internal/rpc/syncAllOrgs   -H 'accept: application/vnd.api+json'   -H 'Content-Type: application/json'   -H 'x-rh-swatch-psk: $SWATCH_SELF_PSK’
```

Comment:
A typo here: need a '/' after v1 and then I think we also need brackets around the env var like '${SWATCH_SELF_PSK}'